### PR TITLE
test(manager): Test manager repair control

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: mgmt_cli_test.py [hunk 1]
```diff
@@ -710,9 +751,10 @@ def _insert_data_while_excluding_each_node(self, total_num_of_rows, keyspace_nam
         for node in self.db_cluster.nodes:
             node.run_nodetool("flush")
 
-    def create_missing_rows_in_cluster(self, create_missing_rows_in_multiple_nodes, keyspace_to_be_repaired=None):
+    def create_missing_rows_in_cluster(self, create_missing_rows_in_multiple_nodes, keyspace_to_be_repaired=None,
+                                       total_num_of_rows=29296872):
```
Comment (fruch) [inline mgmt_cli_test.py:755]: please don't put a default like that, 
put it in the test only, and put a comment where it's taken from how it was calculated, and such.

File: sdcm/mgmt.py [hunk 2]
```diff
@@ -350,6 +350,20 @@ def wait_for_status(self, list_status, check_task_progress=True, timeout=3600, s
...
         return is_status_reached
 
+    def wait_for_percentage(self, minimum_percentage, timeout=3600, step=10):
+        text = f"Waiting until task: {self.id} reaches at least {minimum_percentage}% progress"
+        is_percentage_reached = wait.wait_for(func=self.has_progress_reached_percentage,
+                                              minimum_percentage=minimum_percentage,
+                                              step=step, text=text, timeout=timeout)
+        return is_percentage_reached
+
+    def has_progress_reached_percentage(self, minimum_percentage):
```
Comment (fruch) [inline sdcm/mgmt.py:360]: when adding such utility, please add a docstring, explain what this utility is going todo, and **why** it's needed.

**Comment by fruch** [comment]
@ShlomiBalalis few minor comments,
please update once you had this running, and let's review the outcome togther.

File: mgmt_cli_test.py [hunk 3]
```diff
@@ -332,6 +332,50 @@ def test_repair_intensity_feature_on_multiple_node(self):
...
+        keyspace_to_be_repaired = "keyspace2"
+        if not self.is_cred_file_configured:
+            self.update_config_file()
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
+        mgr_cluster = manager_tool.add_cluster(name=self.CLUSTER_NAME + '_repair_control',
+                                               db_cluster=self.db_cluster,
+                                               auth_token=self.monitors.mgmt_auth_token)
+        self.create_missing_rows_in_cluster(create_missing_rows_in_multiple_nodes=True,
+                                            keyspace_to_be_repaired=keyspace_to_be_repaired,
+                                            total_num_of_rows=292968720)
```
Comment (fruch) [inline mgmt_cli_test.py:361]: still missing a comment of explaining where this 292968720 number is coming from.
it's it % of the whole number of rows ? or what exactly ?

Comment (ShlomiBalalis) [reply mgmt_cli_test.py:363]: Explanation added

**Comment by fruch** [comment]
is this the only test in dtest, that uses SSL ? only this file ?

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**